css/style.css

```diff
@@ -181,6 +181,11 @@ a.ui-button:active,
 	text-align: center;
 }
 
+.properties {
+	font-weight: bold; 
+	color: #3399ff;">
```

Comment:
the closing quote and angled bracket here should be removed.

Reply:
fixed